src/StrawberryShake/Tooling/src/dotnet-graphql/DefaultHttpClientFactory.cs

```diff
@@ -7,7 +8,7 @@ namespace StrawberryShake.Tools
     public class DefaultHttpClientFactory
         : IHttpClientFactory
     {
-        public HttpClient Create(Uri uri, string? token, string? scheme)
+        public HttpClient Create(Uri uri, string? token, string? scheme, Dictionary<string, IEnumerable<string>> customHeaders)
```

Comment:
we have a max line length of 100. So do a line break here.

Reply:
Fixed